src/common/checkbox/checkbox.module.scss

```diff
@@ -2,10 +2,10 @@
 @import 'colours';
 
 .checkbox {
-  --checkbox-dimension: #{units(1)};
+  --checkbox-dimension: 20px;
```

Comment:
are you sure you want those free values after all HDS tokens and variables and stuff?

Reply:
Yeah, I stick to these predefined values for margins, paddings, etc. 
But for the checkbox and radio dimensions, I think these custom values look better.